hub/apps/whats-new/latest-doc-updates.md

```diff
@@ -18,54 +18,50 @@ For the latest Windows Developer Documentation news, or to reach out to us with
+* [PowerToys](/windows/powertoys/): These open source utilities are suggested by and developed with the help of the developer community using Windows. New utilities in the .70 release include:
+  * [Mouse without Borders](/windows/powertoys/mouse-without-borders), for seamless transition between multiple machines from the same keyboard and mouse, sharing clipboard contents and files,  
+  * [Peek](/windows/powertoys/peek), for previewing file content without the need to open multiple apps or interrupt your workflow,
+  * [Paste as Plain Text](/windows/powertoys/paste-as-plain-text), for pasting text from your clipboard, excluding any text-formatting.
 
-* [Windows Terminal](/windows/terminal/): Customize your terminal and running multiple command lines, all set up to their specific preferences. Learn more about how to customize your prompt and the new Tab Tearout feature. 
+* [Windows Terminal](/windows/terminal/): Customize your terminal and running multiple command lines, all set up to their specific preferences. Learn more about how to customize your prompt and the new Tab Tearout feature.
 
-* [Windows Copilot](/windows/dev-environment/): The first PC platform to provide centralized AI assistance and designed to help people easily take action and get things done is coming soon! 
-
-* [Windows on Arm](/windows/arm/overview): We’ve got a new tutorial that can help you update your apps to run natively on Arm. 
+* [Windows Copilot](/windows/dev-environment/): The first PC platform to provide centralized AI assistance and designed to help people easily take action and get things done is coming soon!
 
+* [Windows on Arm](/windows/arm/overview): We’ve got a new tutorial that can help you update your apps to run natively on Arm.
 
 <hr>
 
-
 The following list of topics have also seen significant updates in the past month:
 
 ## Windows App SDK / WinUI
 
 * [WinAppSDK 1.4 Experimental Release Notes](/windows/apps/windows-app-sdk/experimental-channel)
 
-
 ## Code samples, tutorials and Learn Module updates
 
-* Added [examples page for a new Win32 bindlink.h header](/windows/win32/bindlink/bindlink-example). 
-* Added [examples page for a new Win32 hwreqchkapi.h header](/windows/win32/hwreqchkapi/hwreqchk-examples), including four code examples. 
-* Rust: Fixed the code sample in [RSS reader tutorial](/windows/dev-environment/rust/rss-reader-rust-for-windows). 
-* C++/WinRT: Updated [A completion source sample](/windows/uwp/cpp-and-winrt-apis/concurrency-3). 
-
+* Added [examples page for a new Win32 bindlink.h header](/windows/win32/bindlink/bindlink-example).
+* Added [examples page for a new Win32 hwreqchkapi.h header](/windows/win32/hwreqchkapi/hwreqchk-examples), including four code examples.
+* Rust: Fixed the code sample in [RSS reader tutorial](/windows/dev-environment/rust/rss-reader-rust-for-windows).
+* C++/WinRT: Updated [A completion source sample](/windows/uwp/cpp-and-winrt-apis/concurrency-3).
 
 ## Updated content
 
```

Comment:
Is this view parameter needed on line 68? See build suggestion. It should either be removed or add `&preserve-view=true` to the URL if it's needed.

Reply:
Uhm, honestly I have no idea, haha. I just let VS Code (Markdown plug-in) do a quick "Format Document" 😅

Reply:
OK. Probably fine to leave it for now since it's not something you added.